### ISSUE TITLE
How do I add a theme for a new package?

### ISSUE THREAD

**Issue by amno1** [bbatsov/solarized-emacs#450]
I would like to add [this little package](https://github.com/phillord/pabbrev) to that giant macro full of themed packages you have in Solarized. What is the best way to do it? Do I fork solarized, add manually those two faces that the little package uses, and give you a PR, or is there some other method?

Sorry, for the noob question. I just need the red color to melt a bit better in the Solarized Dark. The default one hurts the eyes :).

**Comment by bbatsov** [comment]
> I would like to add [this little package](https://github.com/phillord/pabbrev) to that giant macro full of themed packages you have in Solarized. What is the best way to do it? Do I fork solarized, add manually those two faces that the little package uses, and give you a PR, or is there some other method?

Yep.

**Comment by bbatsov** [comment]
Yep, forking and sending a PR that adds the package's faces is exactly the way to do it - DEV-GUIDE.md walks through it. Closing this, but a PR for pabbrev support would be welcome.
